dojo/tools/sonarqube_api/importer.py

```diff
@@ -143,6 +154,84 @@ def import_issues(self, test):
 
         return items
 
+    def import_hotspots(self, test):
+
+        items = list()
+
+        try:
+            client, config = self.prepare_client(test)
+
+            if config and config.sonarqube_project_key:  # https://github.com/DefectDojo/django-DefectDojo/pull/4676 cases no. 5 and 8
+                component = client.get_project(config.sonarqube_project_key)
+            else:  # https://github.com/DefectDojo/django-DefectDojo/pull/4676 cases no. 2, 4 and 7
+                component = client.find_project(test.engagement.product.name)
+
+            hotspots = client.find_hotspots(component['key'])
+            logging.info('Found {} hotspots for project {}'.format(len(hotspots), component["key"]))
+
+            for hotspot in hotspots:
+                status = hotspot['status']
+
+                if self.is_reviewed(status):
+                    continue
+
+                type = 'SECURITY_HOTSPOT'
+                if len(hotspot['message']) > 513:
+                    title = textwrap.shorten(text=hotspot['message'], width=507)
+                else:
+                    title = hotspot['message']
+                component_key = hotspot['component']
+                line = hotspot.get('line')
+                rule_id = hotspot['key']
+                rule = client.get_hotspot_rule(rule_id)
+                severity = self.convert_sonar_review_priority(hotspot['vulnerabilityProbability'])
```

Comment:
This is not the good value for severity/
According to the SQ documentation:

> Security Hotspots are not assigned severities as it is unknown whether there is truly an underlying vulnerability until they are reviewed.

Security Hotspots are infos to investigate. The data in `vulnerabilityProbability` is the equivalent of the attribute `scanner_confidence` in the current implementation.

Reply:
So, what do you propose to do? Should I leave the severity empty or hardcoded as Info?

Reply:
Simply `Info` because for SonarQube it is info. But you can use this field to populate the confidence attribute.

Reply:
Done